RestCommClient/Classes/RCCallKitProvider.m

```diff
@@ -243,6 +245,18 @@ - (void)presentIncomingCallWithVideo:(BOOL)isVideo{
     [self reportIncomingCallFrom:[self.connection.parameters objectForKey:@"from"] hasVideo:isVideo];
 }
 
+- (void)routeAudioToSpeaker {
+    NSError *error = nil;
+    //https://developer.apple.com/documentation/avfoundation/avaudiosessioncategoryplayandrecord
+    //we need to set category
+    if (![[AVAudioSession sharedInstance] setCategory:AVAudioSessionCategoryPlayAndRecord
```

Comment:
I assume you have left the SipManager.mm AVAudioSession handling untouched, right? Doesn't this mean that `[session setCategory:AVAudioSessionCategoryPlayAndRecord]` might be called twice? Once from here and once from SipManager? Can we check inside SipManager `initializeAudioSession` if it's already set to `AVAudioSessionCategoryPlayAndRecord` and set it again only if it isn't? That way it's cleaner that the audiosession is only called once to set and once to remove.